documentation/asciidoc/topics/proc_disabling_gossip_router_cross_site.adoc

```diff
@@ -0,0 +1,46 @@
+[id='disabling-gossip-router-cross-site_{context}']
+= Disabling local Gossip router and service
+
+[role="_abstract"]
+The {ispn_operator} starts a Gossip router on each site, but you only need a single Gossip router to manage traffic between the {brandname} cluster members.
+You can disable the additional Gossip routers to save resources.
+
+.Prerequisites
+
+* Have multiple working Gossip routers.
++
+You can have an external Gossip router that is not managed by the {ispn_operator}.
+
+.Procedure
+
+. Create an `Infinispan` CR for each {brandname} cluster.
+. Specify the name of the local site with `spec.service.sites.local.name`.
+. Set `false` as the value for the `spec.service.sites.local.discovery.launchGossipRouter`.
```

Comment:
It seems we're setting `spec.service.sites.local.discovery.launchGossipRouter` to `false` in both examples to me :thinking:

Reply:
Yes, I was confused about it https://github.com/infinispan/infinispan-operator/pull/1667/commits/733f955fa2b255ca92f8312e6ac51ddad5865e31#diff-a4d8446815313b75b9b43a4fa99334f650755e298e582973ee7cb8e4b5c234bc